docs/guides/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_dbfs.rst

```diff
@@ -0,0 +1,136 @@
+.. _how_to_guides__configuring_metadata_stores__how_to_configure_an_expectation_store_on_dbfs:
+
+How to configure an Expectation store on DBFS
+=====================================================
+
+
+By default, newly profiled Expectations are stored in JSON format in the ``expectations/`` subdirectory of your ``great_expectations`` folder.  This guide will help you configure a new storage location for Expectations on your filesystem.
+
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
+
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectation Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Determined a new storage location where you would like to store Expectations. This can either be a local path, or a path to a network filesystem.
+
+Steps
+-----
+
+1. **Configure a new folder on your filesystem where Expectations will be stored.**
+
+    Create a new folder where you would like to store your Expectations, and move your existing Expectation files over to the new location. In our case, the name of the Expectations file is ``npi_expectations`` and the path to our new storage location is ``/shared_expectations``.
```

Comment:
Same comment as above, I'd make it explicit when you're talking about local vs DBFS